tests/test_cli_gui/test_run_agent.py

```diff
@@ -71,15 +70,6 @@ def test_create_and_run_agent():
             )
             assert response_add.status_code == 201
 
-        # Add the local connection
-        with patch("aea.cli_gui.app_context.local", True):
-            response_add = app.post(
-                "api/agent/" + agent_id + "/connection",
-                content_type="application/json",
-                data=json.dumps(str(LOCAL_PUBLIC_ID)),
-            )
-            assert response_add.status_code == 201
-
```

Comment:
@DavidMinarsch this made that test flaky

Reply:
Ok, will be removed anyway soon